assets/src/models/blockWaiver.ts

```diff
@@ -51,18 +45,16 @@ export enum BlockWaiverDecoratorStyle {
  */
 export const blockWaiverDecoratorStyle = (
```

Comment:
It feels to me like this function name deserves a change. It feels like it no longer accurately describes what it is providing.